### ISSUE TITLE
Rebase brave omaha to v1.3.36.111

### ISSUE THREAD

**Issue by simonhong** [brave/brave-browser#11904]
https://github.com/google/omaha/releases/tag/v1.3.36.111

**Comment by LaurenWags** [comment]
labelling as `QA/Blocked` due to:
1. Information on logging needed (@mihaiplesa to provide)
2. This will be tested with an RC for 1.34.x, so need to wait until after channel migrations to test this

**Comment by mherrmann** [comment]
Hi, I don't know if this helps but some logging instructions are linked to in https://github.com/brave/brave-core/pull/11096.

**Comment by LaurenWags** [comment]
@brave/legacy_qa this is no longer blocked, 1.34.76 is available: https://github.com/brave/brave-browser/releases/tag/v1.34.76

**Comment by GeetaSarvadnya** [comment]
Verification is inprogress

```

Brave | 1.34.77 Chromium: 97.0.4692.56 (Official Build) (64-bit)
-- | --
Revision | 04da6c66398ca50e603cc236a07dc7dfd3bbc750-refs/branch-heads/4692@{#990}
OS | Windows 10 Version 21H2 (Build 19044.1415)

```

**Comment by stephendonner** [comment]
Verification `IN-PROGRESS` on

Brave | 1.34.77 Chromium: 97.0.4692.56 (Official Build) (64-bit)
--|--
Revision | 04da6c66398ca50e603cc236a07dc7dfd3bbc750-refs/branch-heads/4692@{#990}
OS | Windows 10 Version 20H2 (Build 19042.1415)


**Windows 10 x64 - Physical machine**

<details>
<summary> Case 1. Install different installers from GitHub - PASSED</summary>

- Download `BraveBrowserStandaloneSetup.exe` from Github and install the setup file and confirm brave `1.34.76` is installed successfully and version details are shown correctly in `brave://help`
- Download `BraveBrowserSetup.exe` stub installer from GH and install the setup file and confirm brave `1.33.106` is installed and confirmed latest RC version is pulled from Omaha update server and version details are shown correctly in `brave://help`
- Download `BraveBrowserStandaloneSilentSetup.exe` from GH and install the setup file and confirm brave `1.34.76` is installed successfully also confirm that `StandaloneSilentSetup.exe` do not show any UI or launching Brave.
- Download stub installed from GH or download brave from Chrome/Firefox/IE and install brave and confirm latest RC `1.33.106` is installed successfully, update brave to test channel by installing `BraveBrowserSetup-64-r-test` and confirmed brave   `1.34.77` is updated to test channel without any issues.

1.33.106 - Relaunch | 1.34.77 test channel
------|----------
![11904-1](https://user-images.githubusercontent.com/387249/147996261-6a2daea9-f07b-4c34-b4c1-37494ee8be69.png) | ![11904-2](https://user-images.githubusercontent.com/387249/147996263-702df125-98d7-4ba6-819d-40b8aec8b4bc.png)


</details>

<details>
<summary>Case 2. Re-installing the same version - PASSED</summary>

- Install `BraveBrowserStandaloneSetup.exe` setup file and confirm brave `1.34.76` is installed successfully and version details are shown correctly in `brave://help`. Re-Install standalone installer and confirm `1.34.76` is installed and two browser windows with same version `1.34.76` is observed
- Install `BraveBrowserStandaloneSilentSetup.exe` setup file and confirm brave `1.34.76` is installed successfully and version details are shown correctly in `brave://help`. Re-Install standalone silent installer and confirm `1.34.76` is installed and two browser windows with the same version `1.34.76` is observed
- Install `BraveBrowserSetup.exe` setup file and confirm brave `1.33.106` is installed successfully and version details are shown correctly in `brave://help`. Re-Install stub installer and confirm `1.33.106` is installed and two browser windows with the same version `1.33.106` is observed
</details>

<details>
<summary>Case 3. Installing a new version by older version from different installers - PASSED</summary>

- **BraveBrowserSetup > BraveBrowserStandaloneSetup** : Install BraveBrowserSetup and ensured `1.33.106` is installed and then install `1.34.76` StandaloneSetup on top of it and ensured `1.34.77` is installed successfully.
- **BraveBrowserSetup > BraveBrowserStandaloneSilentSetup**: Install BraveBrowserSetup and ensured `1.33.106` is installed and then install `1.34.77` StandaloneSetup on top of it and ensured `1.34.77` is installed successfully.
- **BraveBrowserStandaloneSetup (1.33.x) > BraveBrowserStandaloneSetup (1.34.x)**: Install `1.33.x` standalone installer and confirm 1.33.x is installed and then install `1.34.77` standalone installer on top of it and ensured `1.34.77` is installed successfully
- **BraveBrowserStandaloneSilentSetup (1.33.x) > BraveBrowserStandaloneSetup (1.34.x)**: Install `1.33.x` standalone silent installer and confirm 1.33.x is installed and then install `1.34.77` standalone installer on top of it and ensured `1.34.77` is installed successfully
</details>

<details>
<summary>Case 4. Install Brave when the Internet is disconnected - PASSED</summary>

- Disconnect the internet and install `BraveBrowserStandaloneSetup.exe` setup file and confirm brave `1.34.77` is installed successfully and version details are shown correctly in `brave://help`
- Disconnect the internet and install `BraveBrowserStandaloneSilentSetup.exe` setup file and confirm brave `1.34.77` is installed successfully and version details are shown correctly in `brave://help`
</details>

<details>
<summary>Case 5. Admin privileges - PASSED</summary>

<details>
<summary>Admin privileges with Yes selection in UAC</summary>

- Confirmed `Standalone setup` installer can be installed successfully with Admin privileges `Yes` and confirmed brave version is shown correctly in brave://help
- Confirmed `Standalone silent setup` installer can be installed successfully with Admin privileges `Yes` and confirmed brave version is shown correctly in brave://help
- Confirmed `Stub installers` can be installed successfully with Admin privileges `Yes` and confirmed brave version is shown correctly in brave://help
- Installed stub installer `BraveBrowserSetup` with admin privileges and confirmed `1.33.106` is installed successfully and then install `BraveBrowserSetup-64-r-test` with admin privileges and confirmed brave is updated to test channel `1.34.77`
- Installed `1.33.106` `BraveBrowserStandaloneSetup` with admin privileges and confirmed 1.33.106 installed successfully and then install `1.34.76`  `BraveBrowserStandaloneSetup` with admin privileges and confirmed 1.34.76 installed

</details>

<details>
<summary>Admin privileges but the user declines admin rights in the UAC prompt</summary>

![11904-3](https://user-images.githubusercontent.com/387249/147998879-f05d039d-b0ee-479a-bc3a-e63dcd5043bb.png)


- Double click on the `BraveBrowserStandaloneSetup` installer and select `No` in the UAC modal popup and select `Yes` in the `...install without administrator privileges. Continue?` prompt and confirmed brave start installing without admin privileges successfully.
- Confirmed `Stub installer` can be installed without Admin privileges and confirmed brave `1.33.106` version details are shown correctly in brave://help.
- Installed `Standalone installer` without Admin privileges and confirmed brave `1.33.106` is installed and then install `1 34.77` installer with Admin privileges and confirmed brave `1.34.77` is installed without admin privileges
</details>

</details>

**Comment by GeetaSarvadnya** [comment]
@mihaiplesa @mherrmann @linhkikuchi - When I install stub installer (BraveBrowserSetup.exe) which pulls the latest RC from Omaha update server and install the 1.33.106 as expected if I install standalone silent installer (BraveBrowserStandaloneSilentSetup.exe_1.34.76) on top of it, brave doesn't get updated to `1.34.76`?

**Comment by simonhong** [comment]
@GeetaSarvadnya IIRC, silent setup is for non-admin install as it doesn't launch any UI.
If system already has brave with admin priv, silent setup will not do anything as it can't update currently installed one.
Based on this, I assume that you installed 1.33.106 with priv mode and then run silent installer?
If you install 1.33.106 w/o admin priv, silent install will update it.

**Comment by GeetaSarvadnya** [comment]
@simonhong Yes, that's correct I have installed a stub installer with Admin privileges and on top of it I was just installing a standalone silent installer (1.34.76). Thanks for the clarification. I have tried installing stub installer without admin privileges and on top of it I installed a standalone silent installer (1.34.76) and brave got updated to `1.34.76` successfully.

Thanks again @simonhong!
